src/apps/pcap/pcap.lua

```diff
@@ -34,11 +34,8 @@ end
 
 PcapWriter = {}
 
-function PcapWriter:new (filename, arg)
-   local conf = arg and config.parse_app_arg(arg) or {}
-   local mode = conf.overwrite and "w+" or "w"
-   local file, errno = io.open(filename, mode)
-   if errno then error(errno) end
+function PcapWriter:new (filename)
+   local file = assert(io.open(filename, "w"))
```

Comment:
The modifier flag was introduced to avoid Snabb returning an error when the pcap file already exists. I initially set it to "w+" always but others preferred to introduce a flag. The way the flag was added doesn't match how other apps work (all of them expect one arg, which is either a string or a table), I didn't know at that time.

Is there any way to keep this behaviour?